include/nbl/builtin/glsl/ext/RadixSort/default_histogram.comp

```diff
@@ -99,24 +103,38 @@ void nbl_glsl_ext_RadixSort_setHistogram(in uint idx, in uint val)
 #define _NBL_GLSL_EXT_RADIXSORT_HISTOGRAM_SET_HISTOGRAM_DEFINED_
 #endif
 
+#ifndef _NBL_GLSL_SCRATCH_SHARED_SIZE_DEFINED_
+#define _NBL_GLSL_SCRATCH_SHARED_SIZE_DEFINED_ (_NBL_GLSL_WORKGROUP_SIZE_ * 2)
+shared uint scratch_shared[_NBL_GLSL_SCRATCH_SHARED_SIZE_DEFINED_];
+#endif
```

Comment:
add an `#else` clause here and check if the user defined `_NBL_GLSL_SCRATCH_SHARED_SIZE_DEFINED_ ` is sufficient